### ISSUE TITLE
Using task view on logiteh mx master 3s open zvirtualdesktop WITH Start menu

### ISSUE THREAD

**Issue by ghost** [mzomparelli/zVirtualDesktop#303]
Using task view on logiteh mx master 3s open zvirtualdesktop WITH Start menu
zvirtualdesktop should not open the start menu. idk how can i fix this.

**Comment by mzomparelli** [comment]
You could change the key combination for DesktopView to something other than Win+Tab. 

Does the Mx Master use g-hub? I have an issue with g-hub mouse that does this with Win-Tab and when I switch to on-board memory it works fine. Not sure what's going on but it's seems like Logitech g-hub issue for me.

**Comment by ghost** [comment]
troubleshooted the issue more. It is your side issue. when having shift+win+tab i don't experience the issue. It has something with the delay between windows and tab.

**Comment by mzomparelli** [comment]
Yeah, I have an issue preventing the WIN key from going up the hook chain after I handle it. I have some code that tries to handle it which you see may not work. I need to look into this more.
